processing/db_handler.py

```diff
@@ -140,6 +140,20 @@ def get_sites_with_updates(self, conn, is_historic=False):
                 lookup[flux_id] = (hh_version, hr_version)
         return lookup
 
+    def get_sites_with_embargo(self, conn):
+        site_ids = []
+
+        # Current embargo is 2 years
```

Comment:
Consider putting the embargo time period in the config file.

Reply:
To be honest this is a little overkill, but I have added it as part of the config